### ISSUE TITLE
Do we need completeness and/or curvature bounds?

### ISSUE THREAD

**Issue by pabryan** [pabryan/rough_metrics_heat_kernel_regularity#4]
We should address the lack of any completeness assumptions. For example, what happens to the heat kernel of the punctured plane? One would expect that some sort of completeness assumption is needed in general for heat kernel estimates, and if we don't address this issue, the experts may well be sceptical of the results since they apply in particular to incomplete, smooth metrics and those with unbounded curvature.

Probably the answer is that we don't have uniform constants but we should at the very least make a note of this fact somewhere.

**Comment by lashputin** [comment]
No. So we should make the setup very clear. We should make clear: 

Every smooth metric, regardless of whether you have curvature bounds or completeness, admits heat kernels. We should say "kernels", because typically, the energy spaces will be H^{1}_0 which is different from H^1. So, for every H^{1}_0 \subset B \subset H^1, for closed subspaces B, we get a heat kernel. When this is something like a bounded domain, the closed subspace can be identified with boundary conditions. I think here, I cite the AKM paper.
